ruby/rails6-shakapacker/app/README.md

```diff
@@ -0,0 +1,25 @@
+# README
+
+## Installation
+
+```
+bundle install
+yarn
+```
+
+## Usage
+
+First, change the revision in `config/appsignal.yml` if you want to test new
+sourcemaps.
```

Comment:
We also need to add the front-end API key to `appsignal_key.env`, yes? If so, it would be nice to mention that here as well, since it's not necessary for the other apps.